### ISSUE TITLE
@fluent/react examples are outdated

### ISSUE THREAD

**Issue by rlr** [projectfluent/fluent.js#424]
In the meantime, is there any real project out there using the latest versions of @fluent/* that I can use as a guide? This whole project seems to be missing a quickstart unless I missed it somewhere? I guess the [ReactBindings](https://github.com/projectfluent/fluent.js/wiki/React-Bindings) wiki page looks useful if it was using the latest version.

An example in typescript for bonus points 😄

**Comment by rlr** [comment]
Also, how do I install the types now? `npm install @types/fluent-react`, for example, used to work but `npm install `@types/@fluent/react` blows up (`Invalid package name "@types/": name can only contain URL-friendly characters`.

**Comment by Gregoor** [comment]
Apart from the package name the code from both the examples and the wiki page should still work as described. Another place to find examples would be the tests, e.g.: https://github.com/projectfluent/fluent.js/blob/master/fluent-react/test/localized_render_test.js

Wrt typings, looks like the naming convention for scoped packages is different there: https://stackoverflow.com/a/47296781/1501916
But from scanning DefinitelyTyped it seems like we haven't changed that yet. @huy-nguyen, I saw you created the typings, do you maybe know more about this?

**Comment by rlr** [comment]
> Apart from the package name the code from both the examples and the wiki page should still work as described. 

Not quite true. `bundle.addMessages(MESSAGES_ALL[locale]);` doesn't work for example. After reading the source code, I learned that now you do `bundle.addResource(new FluentResource(MESSAGES_ALL[locale]));`.

I got it all working now though. Unfortunately, the repo is private for now but I'll share once it is public. Or I'll help update the examples if I get some spare cycles. Thank you!
